src/API/Management/EmailTemplates.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @package Auth0\SDK\API\Management
+ */
+namespace Auth0\SDK\API\Management;
+
+/**
+ * Class EmailTemplates.
+ * Handles requests to the Email Templates endpoint of the v2 Management API.
+ *
+ * @package Auth0\SDK\API\Management\EmailTemplates
+ */
+class EmailTemplates extends GenericResource
+{
+    /**
+     * Get an email template by name.
+     * The email template needs to already exist or the response will be a 404.
+     * An invalid template name will respond with a 400.
+     * See docs @link below for valid names and fields.
+     *
+     * @param string $templateName - the email template name to get.
+     *
+     * @return array
+     *
+     * @throws \Exception - if a 200 response was not returned from the API.
```

Comment:
Be carefull. Some endpoints return success codes different to `200`, i.e. create connection:

![image](https://user-images.githubusercontent.com/3900123/39211241-6df4c6fa-47e1-11e8-8d92-9f420e37e2a9.png)

While this doesn't apply to email-templates, might be worth to check the rest of the entities if this was a copy-paste of that logic/javadocs.

Reply:
I checked before submitting, all are 200 but good eyes 👍